File: backend/src/main/java/xyz/engsmyre/moviedescriptiongame/controller/SuggestionsController.java
package xyz.engsmyre.moviedescriptiongame.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import xyz.engsmyre.moviedescriptiongame.dto.MovieSuggestions;

@Controller
@RequestMapping("/game/suggestion")
public class SuggestionsController {

    @GetMapping
    public MovieSuggestions getMovieSuggestions() {
        return null;
    }
}

File: backend/src/main/kotlin/xyz/engsmyre/moviedescriptiongame/controller/SuggestionsController.kt
package xyz.engsmyre.moviedescriptiongame.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import xyz.engsmyre.moviedescriptiongame.dto.MovieSuggestions
import xyz.engsmyre.moviedescriptiongame.service.SuggestionService

@Controller
@RequestMapping("/game/suggestion")
class SuggestionsController(private val suggestionService: SuggestionService) {
    @GetMapping("")
    fun suggestMovies(searchQuery : String) : MovieSuggestions {
        suggestionService.findSearchResults(searchQuery)
        TODO("Implement me")
    }
}